File: snippets/code/FishHave/bubble_sort.cs
<Type>[] sort(<Type>[] arr) {
    int len = arr.Length;

    for (int i = 0; i < len - 1; i++) {
        for (int j = 0; j < len - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                <Type> temp = arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = temp;
            }
        }
    }

    return arr;
}

File: snippets/code/FishHave/bubble_sort.py
def sort(arr:list):
    len = len(list);
    
    for i in range(len - 1):
        for j in range(len - i - 1):
            if arr[j] > arr[j + 1]:
                temp = arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = temp;
    
    return arr;